src/Network/Freddy.hs

```diff
@@ -5,25 +5,26 @@ import qualified Network.AMQP as AMQP
 import Data.Text (Text)
 import Data.ByteString.Lazy.Char8 (ByteString)
 import qualified Control.Concurrent.BroadcastChan as BC
-import Control.Concurrent (forkIO)
 import qualified Data.UUID as UUID
 import Data.UUID (UUID)
 import System.Random (randomIO)
 import System.Timeout (timeout)
 
 import Network.Freddy.ResultType (ResultType (..), serializeResultType)
+import qualified Network.Freddy.Request as DWP
```

Comment:
a bit hard to name at the moment because there's already on request. This at the moment references DeliverWithResponse. I'm going to do a quite big refacto in the next PR